src/darwin/Framework/Matter.xcodeproj/xcshareddata/xcschemes/Matter Framework Tests.xcscheme

```diff
@@ -38,9 +38,6 @@
                ReferencedContainer = "container:Matter.xcodeproj">
             </BuildableReference>
             <SkippedTests>
-               <Test
-                  Identifier = "MTRRemoteDeviceSampleTests">
-               </Test>
```

Comment:
I tried to find this test that is supposed to came from https://github.com/project-chip/connectedhomeip/commit/153a564928634654576038839f904983ac600fba and I can't tell if this is an extra line (since the header in `MTRXPCListenerSampleTests` uses `MTRRemoteDeviceSampleTests` as a name :/ ) or a file that has not been committed.

Reply:
I don't know....  But in either case, this is not needed and it was confusing me when testing things.  ;)